.github/workflows/publish-cli-bins.yaml

```
@@ -168,11 +168,110 @@ jobs:
+
+            if [ $attempt -eq $max_attempts ]; then
+              echo "Timeout waiting for COPR build to complete"
+              exit 1
+            fi
+          fi
+
+      - name: Download RPMs from COPR
+        run: |
+          build_id="${{ env.build_id }}"
+          mkdir -p copr-rpms
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)